experimental/StandardFiniteFields/src/StandardFiniteFields.jl

```diff
@@ -308,7 +308,7 @@ function standard_monomial_degrees(n::IntegerUnion)::Vector{Int}
   for i in 1:a-1
     append!(res, new)
   end
-  return res
+  return res::Vector{Int}
```

Comment:
I am not opposed to having this (never hurts) but am wondering: is it even necessary?  That would mean `standard_monomial_degrees` is type unstable, I guess?

Reply:
OK there are a bunch of similar situations, I guess it is fair that you only made the "mechanical" change; others can worry (or not) about whether there are deeper type unstabilities here.

Reply:
> OK there are a bunch of similar situations, I guess it is fair that you only made the "mechanical" change; others can worry (or not) about whether there are deeper type unstabilities here.

Indeed, I mostly just moved it. Only in some few situations, e.g. where a constructor is called in the return statement, I omitted it.